src/global/pid_lut/pid_lut.cc

```diff
@@ -97,8 +58,8 @@ void InitPlugin(JApplication* app) {
       "RICHEndcapNLUTPID",
       {
           "EventHeader",
-          "ReconstructedChargedWithFBPIDParticles",
-          "ReconstructedChargedWithFBPIDParticleAssociations",
+          "ReconstructedChargedWithoutFBPIDParticles",
+          "ReconstructedChargedWithoutFBPIDParticleAssociations",
```

Comment:
```suggestion
          "ReconstructedChargedWithoutPIDParticles",
          "ReconstructedChargedWithoutPIDParticleAssociations",
```
I would think, WithoutFBPID is not a thing.